.github/workflows/ci.yml

```diff
@@ -19,41 +19,28 @@ jobs:
         version:
           - 'nightly'
           - '1'
-          - '1.6'
-        os:
-          - ubuntu-latest
-          - macos-latest
-          - windows-latest
-        arch:
-          - x64
+          - '1.6'  # lowest declared julia compat in `Project.toml`
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        arch: [x64]
     steps:
-      - uses: actions/checkout@v2
-      - uses: julia-actions/setup-julia@v1
+      - uses: actions/checkout@v3
+      - uses: julia-actions/setup-julia@latest
```

Comment:
I am not sure that using `latest` for an action is a good idea, as this would change the behavior of this script, whenever a new action is released - but then I also don't have much experience with that.

Reply:
We haven't had any issues with this in repos like `Plots.jl`.